src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexParseError.cs

```diff
@@ -10,10 +10,10 @@ namespace System.Text.RegularExpressions
     /// <remarks>
     /// This information is made available through <see cref="RegexParseException.Error"/>.
     /// </remarks>
-#if REGEXGENERATOR
-    internal
-#else
+#if SYSTEM_TEXT_REGULAREXPRESSIONS
```

Comment:
Curious, why do you prefer a directive for the System.Text.RegularExpressions lib over one for the generator project (the way it was before)?

Reply:
Because if this file is included in some other project, the way it was before these types would end up being included in that project as public.  This way, by default the types would be internal, and that additional project would need to erroneously add a definition for the SYSTEM_TEXT_REGULAREXPRESSIONS constant to get it to be public.  Basically it's flipping the default polarity.